src/sustainable-web-design.js

```diff
@@ -189,12 +204,61 @@ class SustainableWebDesign {
    */
   energyPerVisit(bytes) {
```

Comment:
This function feels far away from the SWD approach on the site. Harder to understand breaking it down into each component than putting it back together. Maybe it's ok. Just for new devs (and me) I constantly think 

> "Oh, this is different, from SWD, why is this?" 

and takes a lot to understand why.